changelog/1.17.3.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "1.17.3"
+description: "Released on 04/08/21"
+---
+
+### Breaking Changes ❗
```

Comment:
```suggestion
### Breaking changes ❗
```

Reply:
We'll need to fix this in the template as well